src/components/ForgotPassword.tsx

```diff
@@ -0,0 +1,356 @@
+          />
+        </div>
+        <div className="w-full h-auto p-6 bg-white rounded-lg shadow border-1 border-[#e4e7ec] flex-col justify-start items-start inline-flex items-center">
+          <div className="text-[#9A0F28] text-center font-semibold text-2xl font-['Kepler Std'] ">
+            {step <= 2
+              ? "Forgot password"
+              : step === 3
+              ? "New password"
+              : step === 4
+              ? "Successful"
+              : "Enter Code"}
+          </div>
+
+          {step === 1 && (
+            <>
+              <div className="w-full text-gray-500 text-center font-normal text-lg font-['Sofia Pro']">
+                Enter your email to receive the 4 digits code.
+              </div>
+              <div className="pt-2 w-full border-b border-1 border-[#E4E7EC]">
+                {" "}
+              </div>
+              <div className="w-full py-8">
+                <TextField
+                  className="w-full"
+                  id="outlined-basic"
+                  label="Email address"
+                  variant="outlined"
+                  onChange={(e) => {
+                    setEmail(e.target.value);
+                    setError("");
+                  }}
+                  error={Boolean(error)}
+                  helperText={error}
+                />
+              </div>
+              <div className="w-full">
+                <button
+                  onClick={handleEmailSubmit}
+                  className="w-full justify-center flex flex-row bg-teal-600 px-4.5 py-2.5 text-white rounded-lg place-items-center text-[14px] font-semibold leading-[20px]"
+                >
+                  Continue
+                </button>
+              </div>
+            </>
+          )}
+          {step === 2 && (
+            <>
+              <div className="w-full text-gray-500 text-center font-normal text-[18px] leading-[28px] font-['Sofia Pro']">
+                Enter your 4 digits code that you received on your email.
+              </div>
+              <div className="pt-2 w-full border-b border-1 border-[#E4E7EC]">
+                {" "}
+              </div>
+              <div className="flex flex-col gap-8 py-8">
+                <div className="flex space-x-2">
+                  {code.map((value, index) => (
+                    <TextField
+                      key={index}
+                      value={value}
+                      onChange={(e) =>
+                        handleCodeChange(e.target.value.slice(-1), index)
+                      }
+                      onKeyDown={(e) => handleKeyDown(e, index)}
+                      inputRef={(el) => (inputs.current[index] = el)}
+                      error={codeError}
+                      className={`w-[50px] h-[56px] text-center text-lg ${
+                        codeError ? "border-rose-600" : "border-gray-300"
+                      } rounded-lg`}
+                      inputProps={{ maxLength: 1 }}
```

Comment:
nit: I know the Figma looks different but let's make the text center for each of the TextFields